app/templates/components/paper-select.hbs

```diff
@@ -42,6 +42,24 @@
     {{/component}}
   {{/menu.trigger}}
 
+  {{#unless hideAllMessages}}
+    <div class="md-errors-spacer">
+      {{#if maxlength}}
+        <div class="md-char-counter">{{renderCharCount}}</div>
+      {{/if}}
+    </div>
+    {{#if isInvalidAndTouched}}
+      <div class="md-input-messages-animation md-auto-hide" ng-messages>
+        {{#each validationErrorMessages as |error index|}}
+          <div id="error-{{inputElementId}}-{{index}}"
+          class="paper-input-error ng-enter ng-enter-active" ng-message>
+            {{error.message}}
+          </div>
+        {{/each}}
+      </div>
+    {{/if}}
+  {{/unless}}
```

Comment:
I think we could benefit from creating a `{{paper-validation-errors` component with this logic.
As I understand, it is shared with paper-input, and possibly others in the future?